src/flow.h

```diff
@@ -346,6 +347,10 @@ typedef struct Flow_
     uint8_t proto;
     uint8_t recursion_level;
     uint16_t vlan_id[2];
+    uint8_t vlan_idx;
+
+    /** Incoming interface */
+    struct LiveDevice_ *livedev;
```

Comment:
I think this should be a const ptr as the livedev should never be changed from the flow

Reply:
Done, this does require casting in the pseudopacket assignment to avoid compiler warnings (`warning: assignment discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]`)